templates/patterns/date_block/pattern-date-block.html.twig

```diff
@@ -11,4 +11,5 @@
   'year': date|date('Y'),
   'date_time': date|date('Y-m-d'),
   'attributes': attributes,
+  'icon_path': bcl_icon_path
```

Comment:
Not sure if we need that, or if it has to be applied elsewhere. I think the bcl-card component, will now expect a path to icons by default.